examples/jsm/loaders/3DMLoader.js

```diff
@@ -809,9 +821,16 @@ function Rhino3dmWorker() {
 				const buffer = message.buffer;
 				libraryPending.then( () => {
 
-					const data = decodeObjects( rhino, buffer );
+					try {
+
+						const data = decodeObjects( rhino, buffer, message.id );
```

Comment:
If you make a `taskId` variable in the worker, you wouldn't need to pass the id in the methods. On line 823 you could do: 
`taskId = message.id`
You'd need to declare taskId around line 792

Reply:
done